azurerm/resource_arm_automation_job_schedule.go

```diff
@@ -0,0 +1,205 @@
+package azurerm
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/Azure/azure-sdk-for-go/services/automation/mgmt/2015-10-31/automation"
+	"github.com/hashicorp/terraform/helper/schema"
+	uuid "github.com/satori/go.uuid"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/tf"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func resourceArmAutomationJobSchedule() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceArmAutomationJobScheduleCreate,
+		Read:   resourceArmAutomationJobScheduleRead,
+		Delete: resourceArmAutomationJobScheduleDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+
+			"resource_group_name": resourceGroupNameSchema(),
+
+			"automation_account_name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
```

Comment:
Could we get some validation on this property?

Reply:
Added validation by adding them to the azure helpers folder. These could also be used on the automation, runbook and schedule resources as they currently have no/out-of-date validation.

Reply:
thanks! definitely will add them to those resources once this has been merged